.github/workflows/RELEASE.yaml

```diff
@@ -69,6 +69,9 @@ jobs:
           git tag -a "${tag}" -m "${message}"
           git push origin "${tag}"
 
+      - name: Install element template bundling dependencies
+        run: sudo apt install jq
+
```

Comment:
I assume you tried that, but `jq` should already be provided in the ubuntu image this runs in. Just wanted to make we don't do this unnecessarily.

Reply:
Is jq provided? I did not find documentation on it on the internets. I could of course double check with a GitHub action.

Reply:
I can confirm that it is. Handy! https://github.com/nikku/testtest-checks/actions/runs/3248960228/jobs/5330799932

Reply:
Will remove this bit.

Reply:
People seem to use this: https://stackoverflow.com/questions/72752290/using-jq-to-dynamically-build-json-for-use-with-github-actions-strategy-matrix

Reply:
Fixed via https://github.com/camunda/connectors-bundle/pull/17/commits/2ec8f7d70ec5b608f542fe4150c39679e3a65109.